tests/quic/testStreams.nim

```diff
@@ -185,23 +187,58 @@ suite "streams":
     await sleepAsync(100.milliseconds) # wait for stream to be closed
 
     let serverStream = await server.incomingStream()
-    let incoming = await serverStream.read()
-    check incoming == message
+    check (await serverStream.read()) == message
 
     await simulation.cancelAndWait()
 
-  asyncTest "closeWrite() prevents further writes":
+  asyncTest "closeWrite() basic test":
     let simulation = simulateNetwork(client, server)
-    let message = @[1'u8, 2'u8, 3'u8]
-
     let clientStream = await client.openStream()
-    await clientStream.write(message)
+    let serverStreamFuture = server.incomingStream()
+    await clientStream.write(@[]) # Activate stream
+    let serverStream = await serverStreamFuture
```

Comment:
Does the order here matter? could this be:
```suggestion
    await clientStream.write(@[]) # Activate stream
    let serverStream = await server.incomingStream()
```

Reply:
fixed in this and other test cases